traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1623,9 +1623,7 @@ sub check_lwp_response_content_length {
 	my $url           = $lwp_response->request->uri;
 
 	if ( !defined($lwp_response->header('Content-Length')) ) {
-		( $log_level >> $panic_level ) && print $log_level_str . " $url did not return a Content-Length header!\n";
-		exit;
-		return 1;
+		return 0; # Content-Length MAY be omitted per HTTP/1.1 RFC 7230, and in fact MUST NOT be included with a 'Transfer-Encoding: Chunked' header, which MUST be accepted by clients.
```

Comment:
Current recommendations are to implement your own `X-` header and use a nice modern hash. Reusing the `Content-MD5` header is not recommended because many clients implemented it (varyingly, leading to trouble) before it was pulled from the spec. I'd either use an `X-Content-SHA256` field or, alternately, a totally separate request that just fetches the hash. (Race conditions about on that approach, though.)